content/docs/user-guide/experiment-management/sharing-experiments.md

```diff
@@ -4,6 +4,11 @@ You can send [live metrics and plots] to [Studio], [push] entire completed
 <abbr>experiments</abbr> (including data, models, and code), and convert an
 experiment into a [persistent] branch or commit in your Git repo.
 
+See the video below for how it all works using the [DVC Extension] for VS Code,
+or keep reading to go deeper.
+
+https://www.youtube.com/watch?v=UMVYjwJtRj0
```

Comment:
[Q] Did you rewatch this video to make sure it has the correct content? Should the link be to [this one](https://www.youtube.com/watch?v=FggmUFBIFw8&t=75s)? Also... can we use a gif instead 😄 😬.

Reply:
> [Q] Did you rewatch this video to make sure it has the correct content? Should the link be to [this one](https://www.youtube.com/watch?v=FggmUFBIFw8&t=75s)?

🤔 I thought this was a good video for here because it goes end to end from adding the token, to pushing experiments, to sharing a studio link.

> [Q] Did you rewatch this video to make sure it has the correct content? Should the link be to [this one](https://www.youtube.com/watch?v=FggmUFBIFw8&t=75s)? Also... can we use a gif instead 😄 😬.

Up to you. My take would be that besides saving you form having to do two versions of product updates, having you talk through it is more helpful and easier to follow.